metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/indexbuilder/ESIndexBuilder.java

```diff
@@ -43,6 +43,8 @@ public class ESIndexBuilder {
   private final int numReplicas;
   private final int numRetries;
 
+  private final int refreshIntervalSeconds;
+
   private static final List<String> SETTINGS_TO_COMPARE = ImmutableList.of("number_of_shards", "number_of_replicas");
```

Comment:
It should also be added here to know if we need to update the index settings.

Reply:
I see this information in the application.yml. Applying suggestion there.

Reply:
Not sure what you mean here, I'm talking about the SETTINGS_TO_COMPARE immutable list which controls the start up check to see whether or not we should update the index settings during bootstrap.

Reply:
Oh, I thought you were asking about the line that I added, basically asking for me to describe when to increase the interval seconds. Let me take a closer look at the SETTINGS_TO_COMPARE

Reply:
Added the setting to this list & included a comment. Good catch here! Thanks!